### DISCUSSION TITLE
タイトル欄のdiffで変更箇所のfont-weightがnormalになる

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#870] [要望・アイデア・不具合]
![Screenshot 2024-06-27 at 19 35 34](https://github.com/increments/qiita-discussions/assets/1148320/d64bd101-946c-48ab-a207-0e3a03e26f78)

編集リクエストでタイトルを修正したのですが、修正した箇所だけfont-weightがnormalになっていて違和感があります。
タイトルはboldっぽいので修正箇所もboldになっている方が自然なのではと思ったのですがいかがでしょうか？

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご報告ありがとうございます！
こちら修正いたします🙇

**Comment by gilly135** [comment]
@JunichiIto 
修正が完了いたしました！
ご報告いただきありがとうございました :pray:

**Reply by JunichiIto** [reply]
@gilly135 修正ありがとうございます！が、まだ修正されていないかも？
再度ご確認よろしくお願いしますー。
https://qiita.com/hattsuan/items/8fb6f083c27d0643dcef/revisions

![Screenshot 2024-07-05 at 21 02 17](https://github.com/increments/qiita-discussions/assets/1148320/be42996e-36dc-4e26-a95d-f0632163dee5)

**Reply by tanaka-asami415** [reply]
@JunichiIto 
ご確認ありがとうございます。
大変失礼いたしました「編集リクエスト」は修正したのですが「編集履歴」を見逃しておりました🙏
他にも同様の箇所がないか確認して修正いたします🙇

**Reply by gilly135** [reply]
@JunichiIto
この度は修正が漏れていて申し訳ありませんでした :bow: 

下記ページのタイトルのdiffのfont-weightを調整いたしました。

- 記事の編集リクエスト
- 記事の編集履歴
- 質問の編集履歴

ご報告いただきありがとうございました！

**Reply by JunichiIto** [reply]
@gilly135 修正されているのを確認しました。ご対応どうもありがとうございました！
